docs/conf.py

```diff
@@ -45,6 +45,7 @@
 # The theme to use for HTML pages
 html_theme = "furo"
 
+html_title = "Python Documentation Community"
```

Comment:
```suggestion
html_title = "Python Docs Community Group"
```

This would be more in-line with the README and charter for group and Editorial Board.